pkg/landscaper/installations/imports/constructor.go

```diff
@@ -270,3 +281,41 @@ func (c *Constructor) templateDataMappings(
 	}
 	return values, nil
 }
+
+// RenderImportExecutions renders the blueprint's ImportExecutions.
+// Has to be called after import construction (c.Construct(...))
+func (c *Constructor) RenderImportExecutions() error {
+	cond := lsv1alpha1helper.GetOrInitCondition(c.Operation.Inst.GetInstallation().Status.Conditions, lsv1alpha1.ValidateImportsCondition)
+
+	templateStateHandler := template.KubernetesStateHandler{
+		KubeClient: c.Operation.Client(),
+		Inst:       c.Operation.Inst.GetInstallation(),
+	}
+	targetResolver := secretresolver.New(c.Operation.Client())
+	tmpl := template.New(gotemplate.New(c.Operation.BlobResolver, templateStateHandler, targetResolver), spiff.New(templateStateHandler))
```

Comment:
Just one question. Why requires a go-templater a blobresolver and a target resolver and a spiff-templater not?

Reply:
I was curious, so I checked:
There seems to be an asymmetry in the available custom functions between gotemplate and spiff:
For gotemplate, there is a `resolve` function, which requires the `blobResolver` and multiple `get*Kubeconfig` functions, which require the `targetResolver`. For spiff, neither of these functions are available, therefore the arguments are not required.

Generally speaking, I think we should keep the custom functions symmetric between all templating implementations. However, since spiff will likely not be used by many people, I don't think that the current state will actually bother anyone. So let's keep it this way until someone complains.